app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsFragment.kt

```diff
@@ -163,7 +163,11 @@ class ChatRoomsFragment : Fragment(), ChatRoomsView {
             /*val diff = async(CommonPool) {
                 DiffUtil.calculateDiff(RoomsDiffCallback(adapter.baseAdapter.dataSet, newDataSet))
             }.await()*/
-
+            if (newDataSet.isEmpty()) {
+                text_no_search.visibility = View.VISIBLE
```

Comment:
Thanks for this PR! 
We are using some extensions function on the app, so next time you can use: 
`view.isVisible = true or false`